integration-test/src/test/scala/com/cloudera/livy/test/JobApiIT.scala

```diff
@@ -157,16 +157,17 @@ class JobApiIT extends BaseIntegrationTestSuite with BeforeAndAfterAll with Logg
     assert(result === "hello")
   }
 
-  test("run scala jobs") {
+  test("run echo jobs") {
```

Comment:
@tc0312 One thing I may ignored, since IT is built against scala 2.10 and which will add test-lib as dependency, so even if we built two versions of test-lib, we could still use test-lib_2.10, unless we change IT to build against 2.11. So I think it is not necessary to build two versions of test-lib.